notebooks/02-quality-flags/TIKEBook_Lesson2.ipynb

```diff
@@ -0,0 +1,780 @@
+{
```

Comment:
<p>We should justify why we're only choosing sector 16 instead of both, since both 15 and 16 were mentioned above.</p><p> </p><p>Unrelated to previous sentence: am I crazy or is the normalized flux now the default column for the lc object? I thought it used to be pdcsap_flux for Kepler (but now for some reason when in lk it's kspsap_flux) and that's what the tutorial shows if you don't specify normalize=true (https://docs.lightkurve.org/tutorials/1-getting-started/using-light-curve-file-products.html#Using-Light-Curve-Files-with-Lightkurve)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/project-tikebook/pull/19/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>